### ISSUE TITLE
Any Plan for intergration with vite2?

### ISSUE THREAD

**Issue by cyfung1031** [gridsome/gridsome#1576]
Vite2 is fast, but not JAMstack.

Any plan for the intergration with that great tools developed by Evan You ?

https://github.com/vitejs/vite/discussions/5200

**Comment by hjvedvik** [comment]
@cyfung1031 Yes, we are working on it :) We will probably come with some more information later this autumn.

**Comment by Simonl9l** [comment]
@hjvedvik any word on this, being now October 2022...

**Comment by melenaos** [comment]
Is there any news on that? Can I help with testing?
